astropy/utils/tests/test_data.py

```diff
@@ -48,13 +45,28 @@ def test_download_nocache():
 
 @pytest.mark.remote_data('astropy')
 def test_download_parallel():
-    from ..data import download_files_in_parallel
-
-    fnout = download_files_in_parallel(
-        ['http://data.astropy.org',
-         'https://astropy.stsci.edu/data/intersphinx/README'])
+    import shelve
+    from ..data import (download_file, download_files_in_parallel,
+                        _get_download_cache_locs, get_cached_urls)
+
+    main_url = 'http://data.astropy.org/intersphinx/README'
+    mirror_url = 'http://www.astropy.org/astropy-data/intersphinx/README'
```

Comment:
is astropy.org really a mirror, isn't it more fool proof to link directly to github as the mirror here?

Reply:
It is the mirror listed in `astropy.utils.data.conf`. I could use GitHub but it would be inconsistent with actual config -- @astrofrog ?

Reply:
oh, OK, then never mind my comment.